src/main/java/com/prefect/office/record/management/appl/facade/prefect/communityservice/CommunityServiceFacade.java

```diff
@@ -30,7 +31,7 @@ public interface CommunityServiceFacade {
      * @param studentId is the id of the Student that has a Community Service History
      * @return A list of all community service records of the specific student
      */
-    List<CommunityService> getAllCsByStudentId(String studentId);
+    List<CommunityService> getAllCsByStudentId(Student studentId);
```

Comment:
function name is getAllCSByStudentId, why use Student as parameter?

Reply:
sir ginaya ko lang po yung ginawa ko po sa getAllOffenseByStudentId since I thought po na it will be more flexible in getting the other information of the student po, and ginaya ko po kasi yung Data type sa model na Student student hehe, palitan ko po ba?